pages/events.js

```diff
@@ -0,0 +1,48 @@
+import Head from "next/head";
+import Link from "next/link";
+import { useState } from "react";
+import EventPreview from "../components/events/EventPreview";
+import app from "../config/app.json";
+
+export async function getServerSideProps(context) {
+  let users = [];
+  try {
+    const res = await fetch(`${app.baseUrl}/api/users`);
+    users = await res.json();
+  } catch (e) {
+    console.log("ERROR search users", e);
+  }
+
+  return {
+    props: { users },
+  };
+}
+
+export default function Events({ users }) {
+
+  return (
+    <>
+      <Head>
+        <title>LinkFree Check Events</title>
+        <meta name="description" content="Search LinkFree user directory" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <div className="flex flex-col px-6 align-center">
+        <ul>
+          {users.map((user) => {
```

Comment:
Note for a future issue/pr, the list should be in date order, not user order. My suggestion would be to do this on the server side and here just display the list of events.

This is not blocking the PR though.

Reply:
Sure sir, I'll look into it

Reply:
If you get time that would be great 👍 , but it can come in another issue and PR. I will merge this soon